src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -119,9 +119,87 @@ def map_to_bools(inputs: Any) -> Any:
 # ndarrays in an element by element fashion with a lambda which performs
 # the same function on the input type. These functions are called from
 # the __array_ufunc__ function
+#
+# Functions which are not implemented are explicitly listed here as
+# well to provide the reader with an understanding of the current
+# breadth. If a new ufunc is defined by numpy the absence of that
+# function from this list also implies that is it NotImplemented
+#
+# Numpy ufunc Reference
 # https://numpy.org/doc/stable/reference/ufuncs.html
 numpy_to_pandas_universal_func_map = {
+    # Math functions
     "add": lambda obj, inputs, kwargs: obj.__add__(*inputs, **kwargs),
+    "subtract": lambda obj, inputs, kwargs: obj.__sub__(*inputs, **kwargs),
```

Comment:
Good catch. I think I should fix it in this PR. I think the kwargs here should be mostly ignored possibly.

Reply:
Yeah. I believe the right behavior should be to ignore kwargs, but maybe throw an error if out is used ( since that type of operation would not be allowed w/ snowpandas.

Reply:
BigFrames will return NotImplemented if kwargs is > 0. PySpark Pandas will return NotImplemented if out is anything other than None and also ignores kwargs.

Overall this is a really good catch.

Reply:
@sfc-gh-mvashishtha take a look at the solution I pushed. let me know if it makes sense.

Reply:
SGTM